rocky/rocky/views/ooi_view.py

```diff
@@ -146,29 +145,29 @@ def form_valid(self, form):
             form.add_error("__all__", str(exception))
             return self.form_invalid(form)
 
-    def get_success_url(self, ooi) -> str:
-        return get_ooi_url("ooi_detail", ooi.primary_key, self.organization.code)
-
     def get_readonly_fields(self) -> List:
         if not hasattr(self, "ooi"):
             return []
 
         return self.ooi._natural_key_attrs
 
 
-@class_view_decorator(otp_required)
-class BaseDeleteOOIView(SingleOOIMixin, TemplateView):
-    def setup(self, request, *args, **kwargs):
-        super().setup(request, *args, **kwargs)
-        self.octopoes_api_connector = self.octopoes_api_connector
-
-    def delete(self, request):
-        self.octopoes_api_connector.delete(self.ooi.reference)
-        return HttpResponseRedirect(self.get_success_url())
-
-    # Add support for browsers which only accept GET and POST for now.
-    def post(self, request, **kwargs):
-        return self.delete(request)
-
-    def get_success_url(self):
-        return reverse_lazy("ooi_list", kwargs={"organization_code": self.organization.code})
+class OOIBreadcrumbsMixin(BreadcrumbsMixin, BaseOOIDetailView):
```

Comment:
This actually was merely a move of `OOIBreadcrumbsMixin` and a removal of `BaseDeleteOOIView`.